### ISSUE TITLE
Understanding WebSADL git capabilities

### ISSUE THREAD

**Issue by crapo** [SemanticApplicationDesignLanguage/sadl#464]
@kittaakos , you have previously pointed me to this eclipse-theia issue (https://github.com/eclipse-theia/theia/issues/1037#issuecomment-376180725/) starting where you wrote "Alternatively, here is a complete guide to support multiple GitHub accounts with a single Theia backend....". I have been trying to understand this and follow the instructions to demonstrate the approach with WebSADL but have not been successful. Should this approach work with WebSADL? Are these cmds to be done in the WebSADL terminal window or in a cmd session (e.g., PuTTY) on the server? Any clarification that you can give on how to apply these instructions successfully would be appreciated.

**Comment by kittaakos** [comment]
> Should this approach work with WebSADL?

Yes, the example has been verified, I went through the steps when I was writing the doc. It should work.

> Are these cmds to be done in the WebSADL terminal window or in a cmd session (e.g., PuTTY) on the server?

They are the same. If you access the terminal from the web-app (webSADL from the browser), you are accessing the server's terminal.

> Any clarification that you can give on how to apply these instructions successfully would be appreciated.

What's wrong with the instructions, do you see an error when you execute them, or does it work incorrectly?

**Comment by GEGlobalResearch** [comment]
John Interrante and I spent some time this morning going through these instructions and I think we understand better. Just to confirm:
1. Each user ends up with their own folder under the workspace folder. Their personal clone of arepo is under this folder.
2. Their generated RSA keys will be stored in their personal folder.
3. All uses will be able to see and access files in other users' folders. In particular, they will have access to other users' private github RSA keys.

**Comment by kittaakos** [comment]
> John Interrante and I spent some time this morning going through these instructions and I think we understand better.

Excellent!

> 1. Each user ends up with their own folder under the workspace folder. Their personal clone of arepo is under this folder.

Yes, each user has her own folder containing a local git repository cloned with SSH.

> 2\. Their generated RSA keys will be stored in their personal folder configured with SSH.

I put the keys to the user's home folder (`~/.ssh`). I have not tried putting it into the user's Theia workspace folder, if you want that way and it works it's great.

> 3\. All uses will be able to see and access files in other users' folders. In particular, they will have access to other users' private github RSA keys.

Yes. In fact, users can access anything on the server from the terminal.
